gytheio-content-handlers/pom.xml

```diff
@@ -17,4 +17,27 @@
         <module>gytheio-content-handler-tempfile</module>
     </modules>
 
+    <dependencies>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.11</version>
```

Comment:
Should we pull versions up to a property ?

Why 2.2.11 (rather than 2.2.12 or higher) ?

Reply:
Not sure about the properties as they are only in the parent version and not re-used but I can easily do that.

I've used those versions are those libs were present in JDK8 and those were [scheduled for removal](http://openjdk.java.net/jeps/320).

Reply:
FYI: Internal Java 11 doc mentions 2.3.0.